### PR TITLE
Use subprocess for rms

### PR DESCRIPTION
**Pull request by ManInFez**
Closes https://github.com/equinor/libres/pull/1030

use wrapper script for rms if available

Should fix the RMS issues by executing rms through a komodo-disable-script defined in the rms_config.yml file

### CODE REVIEW

**Comment by lars-petter-hauge** [comment]
Throwing os.fork out and use subprocess instead, sounds good to me! Unsure how much tested the RMS stuff in libres is though

File: python/res/fm/rms/rms_run.py [hunk 1]
```diff
@@ -155,7 +145,10 @@ def wait(self, child_pid):
             )
 
     def exec_rms(self, exec_env):
-        args = [
+        args = []
+        if self.config.wrapper is not None:
+            args = [self.config.wrapper]
```
Comment (sondreso) [inline python/res/fm/rms/rms_run.py]: Maybe like this? 
```suggestion
        args = [] if self.config.wrapper is None else [self.config.wrapper]
```

File: python/res/fm/rms/rms_run.py [hunk 2]
```diff
@@ -155,7 +145,10 @@ def wait(self, child_pid):
...
+        args += [
```
Comment (sondreso) [inline python/res/fm/rms/rms_run.py]: or like this:
```suggestion
        args = []
        if self.config.wrapper is not None:
            args.append(self.config.wrapper)
        args.extend([
```

NB: Missing ) at the end

**Comment by sondreso** [comment]
I think the commit message is a little misleading since the wrapper functionality is introduced in addition to the fork change.  Would it be possible to split them into two commits and/or explain why the wrapper is introduced in commit body?

**Review by sondreso — APPROVED** [review]
LGTM!

**Pull request merged**